src/Symfony/Component/Intl/Collator/Collator.php

```diff
@@ -106,7 +106,7 @@ public static function create($locale)
      *
      * @return bool True on success or false on failure
      */
-    public function asort(&$array, $sortFlag = self::SORT_REGULAR)
+    public function asort(&$array, int $sortFlag = self::SORT_REGULAR)
```

Comment:
Can we use the type-hint for the $array variable?

```php
public function asort(array &$array, int $sortFlag = self::SORT_REGULAR)
{
    //
}
```

Reply:
yep, missed it, thanks